docs/internals/details.rst

```diff
@@ -0,0 +1,46 @@
+Implementation Details
+======================
+
+
+.. _log_density_propto:
+
+Speed of the ``propto`` argument
+--------------------------------
+
+The log density function provided by a Stan model has
+the ability drop additive constants from the calculation.
+This is indicated by the ``propto`` ("``prop``\ortional  ``to``")
+argument to function.
+
+If you are using an application such as an MCMC algorithm which requires
+gradients and only needs the log density up to a proportion, setting
+``propto=True`` will be at least as fast as setting ``propto=False``
```

Comment:
Aren't calculations at most as fast with `propto = True`?  `propto = True` won't be faster, right?  Just maybe as fast or less fast.

Reply:
No.  If we are evaluating double values, then `propto=True` requires evaluating autodiff, whereas `propto=False` requires evaluating normalizing constants, but only with `double` inputs.  So it can go either way.